pkg/status/status_server.go

```diff
@@ -236,6 +238,18 @@ func (s *Server) loggersHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (s *Server) accesslogHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
we should limit only POST is supported here

Reply:
Done